handbook/engineering/hiring/software-engineer-campaigns.md

```diff
@@ -0,0 +1,75 @@
+- We’re in the early stages of creating this product, and have a lot of [autonomy and agency](https://about.sourcegraph.com/company/values#high-agency). As a result, we all have significant impact on the shape and direction of Campaigns. We can see our own fingerprints all over, making it all the more delightful when customers love our work.
+- We have all sorts of interesting challenges and opportunities for growth. Frontend and backend coding, architecture and design, authoring RFCs, testing, demoing to customers, collecting and incorporating feedback, supporting our customers… we do it all.
+- Our technical challenges are varied and fun: from git diff parsing to TypeScript, to efficiently querying REST/GraphQL APIs without overloading a code host, to building a “declarative distributed system” that reconciles state across code hosts, to designing the DSL that is our campaign spec, to building terminal emulation into our tests, it does not get dull around here.
+- We have so many exciting ideas on our roadmap! Better changeset sorting/filtering/management, automated labels/reviewers/merging, bringing the power of campaigns to monorepos, server-side execution, and more. We are still at the beginning of this journey, with so much cool work in front of us.
+- We’re a team full of funny, highly-skilled, empathetic developers who are all great communicators. Especially the manager, who is just all-around wonderful (and, it should be noted, is the one writing this).
+
+## What we need from you
+
+We are looking for a full-stack engineer (or a front-end engineer with full-stack aspirations, eager to learn) who has strong fundamentals in good software development techniques and best practices, and who has built and deployed production-ready software that delivered value to customers.
+
+Most of all, we’re looking for an engineer with whole-product concern and accountability. Someone who understands that success is measured by our customers, and that everything that impacts customer satisfaction is part of your job. If working directly with sales, marketing, support, and our customers feels like “that’s not my job,” then this is not your job. On the other hand, if you’ve shipped a product, beginning to end (for work or on the side, software or otherwise), we want to hear from you.
+
+Beyond that, we’re looking for the following:
+
+- Experience creating single page applications using React or similar component-based frontend frameworks.
+- Skilled at writing clean, reusable CSS and semantic HTML to implement visual designs.
+- Familiarity with TypeScript or a different typed programming language.
+- Experience in at least one server side language, such as Go, Java, Python, PHP, or Ruby.
```

Comment:
Maybe some hint that this doesn't need to be professional experience (at least I understood that's out understanding of 'experience' here)

Reply:
Agree with both points:

1. We don't require "3+ years of Go in a professional setting", but we do require experience in the language (i.e. it's fine if you want to switch from using Go as your side-project language to using it fulltime). But I wonder which applications we'll get if we loosen the constraints (maybe that's a premature worry though)
2. I think experience with HTTP and web technology (REST, GraphQL, etc.) are equally important.